test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetVerifyTests.cs

```diff
@@ -46,7 +46,7 @@ public async Task Verify_UnSignedPackage_Fails()
             }
         }
 
-        [CIOnlyFact]
+        [PlatformFact(Platform.Windows, Platform.Linux)] // https://github.com/NuGet/Home/issues/11178
```

Comment:
Wouldn't the tests fail locally now?

Reply:
Or rather they'd run locally.

Reply:
There is no existing CIOnly + Platform specific xunit attribute we could leverage. So I guess we need to create an attribute to combine CIOnly and Platform specific attribute if we want to skip on Mac and make it runs on CI only.
But those tests are able to run locally.